apps/synapse-portal-framework/src/components/elportal/ELGettingStarted.tsx

```diff
@@ -0,0 +1,139 @@
+import { Box, Button, Typography } from '@mui/material'
+import React from 'react'
+import exploreIcon from './assets/explore_icon.png'
+import uncoverIcon from './assets/uncover_icon.png'
+import accessIcon from './assets/access_icon.png'
+
+const ELGettingStarted: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: 'primary.main',
+        display: 'grid',
+        p: '80px',
+        gridTemplateColumns: {
+          xs: '100%',
+          sm: '50% 50%',
+          md: '25% 25% 25% 25%',
+        },
+        gap: '10px',
+        gridAutoRows: '1fr',
+      }}
+    >
+      <Box
+        sx={{
+          width: '100%',
+          height: '100%',
+          borderTop: '3px solid #ffffff88',
+          py: '20px',
+        }}
+      >
+        <Typography
+          variant="headline2"
+          sx={{
+            width: '100%',
+            color: 'white',
+            fontFamily: "'Merriweather', serif",
+            fontSize: '24px',
+          }}
+        >
+          Getting Started
+        </Typography>
+        <Typography
+          variant="body1"
+          sx={{
+            maxWidth: '100%',
+            my: '16px',
+            color: 'white',
+            fontStyle: 'italic',
+            fontSize: '13px',
+            lineHeight: '20px',
+          }}
+        >
+          We provide all the help you need for navigating the portal and
+          accelerating your research.
+        </Typography>
+        <Button
+          variant="contained"
+          href="https://help.eliteportal.org/help/"
```

Comment:
Since this goes to another page, it seems like it should open in a new tab
```suggestion
          href="https://help.eliteportal.org/help/"
          rel="noopener noreferrer"
          target="_blank"
```